### PR TITLE
Replace another deprecated Loggers method with LogManager.

### PR DESCRIPTION
**Pull request by pratiksanglikar**
Dropping another deprecated `Loggers` method to be replaced with `LogManager`.

### CODE REVIEW

**Comment by jasontedor** [comment]
Would you please give you PR a more descriptive title?

**Comment by pratiksanglikar** [comment]
This PR is to replace another deprecated Loggers method with LogManager.
It is part of commits to close #32174 .

**Comment by atorok** [comment]
@elasticmachine test this please

**Comment by nik9000** [comment]
@pratiksanglikar, it looks like this conflicted a little with #34606 which I saw first. Could you merge master?

**Review by nik9000 — APPROVED** [review]
Other than the merge conflicts this looks great! Thanks for doing it @pratiksanglikar!

**Comment by pratiksanglikar** [comment]
Hi @nik9000 , I've resolved the conflicts. How can I merge the PR?

**Comment by nik9000** [comment]
You need one of the maintainers to merge the PR. In this case, that is me. I'll give it another look and then kick off a CI build for it and merge it when I can.

File: server/src/main/java/org/elasticsearch/ExceptionsHelper.java
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/ExceptionsHelper.java]: Could you switch this back? It isn't related to your other changes and I don't think it is a change we want anyway.

File: test/framework/src/main/java/org/elasticsearch/test/InternalTestCluster.java
Comment (nik9000) [inline test/framework/src/main/java/org/elasticsearch/test/InternalTestCluster.java]: Could you revert this line removal? I don't mind changing how imports are shaped but I don't think we should do it in files that we're not otherwise editing.

File: test/framework/src/main/java/org/elasticsearch/test/TestCluster.java
Comment (nik9000) [inline test/framework/src/main/java/org/elasticsearch/test/TestCluster.java]: Same here.

File: test/framework/src/main/java/org/elasticsearch/test/disruption/NetworkDisruption.java
Comment (nik9000) [inline test/framework/src/main/java/org/elasticsearch/test/disruption/NetworkDisruption.java]: Same here.

File: test/framework/src/main/java/org/elasticsearch/test/junit/listeners/ReproduceInfoPrinter.java
Comment (nik9000) [inline test/framework/src/main/java/org/elasticsearch/test/junit/listeners/ReproduceInfoPrinter.java]: Same here.

**Review by nik9000 — CHANGES_REQUESTED** [review]
I've left a couple of small requests for changes in a few files. Could you make those?

Thanks for doing this. I've been meaning to clean these up and it is great to have some help!

**Comment by nik9000** [comment]
@elasticsmachine, test this please.

**Comment by pratiksanglikar** [comment]
Hi @nik9000 , I've made the requested changes and resolved the conflicts!

**Review by nik9000 — APPROVED** [review]
Looks good to me. I'm adding this to my list of things to merge today.

**Comment by nik9000** [comment]
@elasticmachine, test this please.

**Comment by nik9000** [comment]
Ah! There is a checkstyle failure:
```
[ant:checkstyle] [ERROR] /home/manybubbles/Code/Elastic/Elasticsearch/elasticsearch/test/framework/src/main/java/org/elasticsearch/test/store/MockFSIndexStore.java:22:8: Unused import - org.apache.logging.log4j.LogManager. [UnusedImports]
```

Could you at it @pratiksanglikar? Something like running `./gradlew checkstyle` should run checkstyle on everything and won't take *forever*. I mean, it'll take a while, but not forever.

**Comment by pratiksanglikar** [comment]
@nik9000 Please merge the PR!

**Comment by nik9000** [comment]
@elasticmachine, test this please

**Comment by nik9000** [comment]
@elasticmachine, test this please

**Comment by nik9000** [comment]
@pratiksanglikar, I resolved another merge conflict by pushing to your branch. I'm running tests now just to be super sure.

**Comment by nik9000** [comment]
OK! Passing build! Wonderful! I'll merge this when the [intake build](https://elasticsearch-ci.elastic.co/view/Elasticsearch/job/elastic+elasticsearch+master+intake/) goes back to green. It ought to do that soon.

**Comment by nik9000** [comment]
@pratiksanglikar, I've merged to master and am backporting to 6.x now. Thanks!

**Comment by nik9000** [comment]
The backport build passed for me late last night so I pushed it this morning. Thanks again @pratiksanglikar!

**Comment by nik9000** [comment]
>  I'm excited to take up the next ticket!

Thanks for keeping up with it! It is really nice to have help with these clean ups!

If you want a similar one I'm going to try and do the same sort of thing on #34488. It isn't ready yet, but I should get to it tomorrow.

**Pull request merged**
